docs/book/walrus-sites/commands.md

```diff
@@ -7,8 +7,58 @@ In general, the `--help` flag is your friend, you can add it to get further deta
 CLI (`site-builder --help`) or individual commands (e.g. `site-builder update --help`).
 ```
 
+## `deploy`
+
+The `deploy` command is the primary and recommended command for managing your Walrus Site on Sui.
+The command takes a directory as input and creates a new Walrus Site from the
+resources contained within, and on subsequent calls, it updates the Existing Site.
+
+### Usage
+
+As shown by the command's help information, the typical usage is:
+
+``` sh
+site-builder deploy [OPTIONS] --epochs <EPOCHS> <DIRECTORY>
+```
+
+The `deploy` command determines whether to publish or update based on the presence of a `site_object_id`
+field in the `ws-resources.json` file. [specifying headers and routing](./routing.md).
+
+```admonish info
+The `site_object_id` field is automatically set by the `deploy`
+command, when deploying a new Site, so there is no need for manually tracking the site's object ID.
+```
+
+```admonish note
+The wallet you are using to update an existing Site must be the *owner* of the Walrus Site object to be able
+to update it.
+```
+
+The `--epochs` flag specifies the number of epochs for which the site's resources will be stored
+on Walrus (e.g., 10). You can also use max to store for the maximum allowed duration.
+
+```admonish warning
+The `--epochs` flag is required & it's value must be greater than 0.
+```
+
+```admonish danger title="Epoch duration on Walrus Testnet"
+On Walrus Testnet, the epoch duration is **two days**. Therefore, consider storing your site for a
+large number of epochs if you want to make it available for the following months! The maximum
+duration is set to 183 epochs (corresponding to one year).
+```
```

Comment:
This is no longer correct, with Testnet v3 one epoch is 1 day and one can store a site for 53 epochs. Also, we should probably talk about Mainnet here (especially as we no longer have a public sites portal for Testnet).